tests/Integration/Behaviour/Features/Context/Domain/StoreFeatureContext.php

```diff
@@ -127,6 +205,46 @@ public function isStoreToggledWithReference(string $reference, string $status):
         Assert::assertEquals((bool) $storeUpdated->isActive(), $isEnabled);
     }
 
+    private function mapToEditableStore(int $storeId, array $data): StoreForEditing
+    {
+        return new StoreForEditing(
+            $storeId,
+            true,
+            $this->getShopIdsByReferences($data['shops']),
+            Country::getIdByName($this->getDefaultLangId(), $data['country']),
+            $data['postcode'],
+            $data['city'],
+            $data['state'] ?? null,
+            $data['latitude'] ?? null,
+            $data['longitude'] ?? null,
+            $data['phone'] ?? null,
+            $data['fax'] ?? null,
+            $data['email'] ?? null,
+            $data['name'] ?? [],
+            $data['address1'] ?? [],
+            $data['address2'] ?? [],
+            $data['hours'] ?? [],
+            $data['note'] ?? []
+        );
+    }
+
+    /**
+     * @param string $shopReferencesAsString
+     *
+     * @return int[]
+     */
+    private function getShopIdsByReferences(string $shopReferencesAsString): array
```

Comment:
shop or store?
I see this different naming, is it the same or different thing?

Reply:
Different things, we cast here the shops array string (i.e `'[shop1, shop2]'`) into a real array.